Light Vox Engine/GameTime.cpp

```diff
@@ -19,13 +19,17 @@ void GameTime::ReleaseInstance()
 void GameTime::UpdateTimer()
 {
     __int64 now;
+#if defined(_WIN32) || defined(_WIN64)
     QueryPerformanceCounter( (LARGE_INTEGER*) &now );
+#else
+    static_assert( false, "Unimplemented alternative to 'QueryPerformanceFrequency'" );
```

Comment:
Does this have to be an `static_assert` ?

Reply:
Yes because QueryPerformanceCounter only exists in Windows.h